src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+import java.util.Random;
+public class Lottery {
+    public static Ball getRandomBall(){
+        Ball RandomBall = new Ball();
+        RandomBall.setColor(ColorSupplier.getRandomColor());
+        int RandomIndex = new Random().nextInt(101);
+        RandomBall.setNumber(RandomIndex);
+        return RandomBall;
```

Comment:
You can directly return the new `Ball` object without assigning it to a temporary variable, which would be more concise and clear.